sql/gen-sql-config-docs.py

```diff
@@ -103,6 +103,13 @@ def generate_sql_configs_table_html(sql_configs, path):
                     )
                 )
 
+            if config.name == "spark.sql.files.ignoreInvalidPartitionPaths":
+                description = config.description.replace("<", "&lt;").replace(">", "&gt;")
+            elif config.name == "spark.sql.hive.quoteHiveStructFieldName":
+                description = config.description.replace("<", "&lt;").replace(">", "&gt;").replace("`", "&#96;")
```

Comment:
shall we do it for all the configs? It's hard to believe that people do want to use HTML tags in the config doc.

Reply:
We have many <code></code> usages, I guess.

Reply:
ah I see